install_requirements.txt

```diff
@@ -0,0 +1,15 @@
+pyyaml
+sqlparse>=0.4.4
+Deprecated
+pyodbc
+importlib-metadata
+azure-cosmos
+timezonefinder==6.0.2
+more_itertools
+pycountry
+databricks-sdk
+# Numpy does not support 3.8 if above 1.24
+numpy<=1.24;python_version<="3.8"
```

Comment:
If we drop 9.3 LTS support, we can probably remove these requirements altogether, by simply requiring Python > 3.8? Just food for thought.

Reply:
Good to know: The databricks-sdk has some restrictions on the request package: `requests>=2.28.1,<3`. I know that conflicted also with some other packages. Just for the ones who are going to update these dependencies in the future.

Reply:
Noted. Resolving this comment.